### PR TITLE
[extensions/openai] Fix error when preparing cache for embedding models

### PR DESCRIPTION
**Pull request by wangcx18**
Address [this error](https://github.com/oobabooga/text-generation-webui/pull/3953#issuecomment-1724755589).
## Checklist:

- [x] I have read the [Contributing guidelines](https://github.com/oobabooga/text-generation-webui/wiki/Contributing-guidelines).

### CODE REVIEW

**Comment by noneabove1182** [comment]
Can you elaborate on the comment you added to the readme? Just curious what it applies to

**Comment by wangcx18** [comment]
@noneabove1182 Sure! Does it make sense to you this time?

**Comment by noneabove1182** [comment]
Ahhh yes it does, that makes it much clearer, thank you!

**Comment by noneabove1182** [comment]
Any chance we can get this merged?

**Comment by wangcx18** [comment]
Hello! @oobabooga @matatonic I believe this PR can be merged quickly. Would you mind giving it a quick review?

File: extensions/openai/cache_embedding_model.py [hunk 1]
```diff
@@ -6,7 +6,6 @@
 import os
 
 import sentence_transformers
-from extensions.openai.script import params
 
-st_model = os.environ.get("OPENEDAI_EMBEDDING_MODEL", params.get('embedding_model', 'all-mpnet-base-v2'))
+st_model = os.environ["OPENEDAI_EMBEDDING_MODEL"] if "OPENEDAI_EMBEDDING_MODEL" in os.environ else "all-mpnet-base-v2"
```
Comment (oobabooga) [inline extensions/openai/cache_embedding_model.py:10]: I think that it would be better to keep the `.get` syntax here

Comment (wangcx18) [reply extensions/openai/cache_embedding_model.py]: OK

**Pull request merged**